set.mm

```diff
@@ -57019,6 +57019,14 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
     ( wfo wa wceq wfn fofn fndmu syl2an crn forn sylan9req jca ) ABEFZCDEFZGACH
     ZBDHQEAIECISRABEJCDEJACEKLQRBEMDABENCDENOP $.
 
+  $( A function is a function onto the image of its domain.  (Contributed by
+     AV, 1-Dec-2022.) $)
+  fimadmfo $p |- ( F : A --> B -> F : A -onto-> ( F " A ) ) $=
```

Comment:
Do we have in set.mm
```
$p |- ( F : A --> B -> F : A -onto-> ran F ) $=
```
? If not, it's worth adding in a future PR.

Reply:
We have `dffn4 $p |- ( F Fn A <-> F : A -onto-> ran F )`. Your theorem follows from this together with ~ffn.

Reply:
Thanks. You be the judge as to adding it in a future PR.